config/routes.rb

```diff
@@ -2,6 +2,7 @@
   mount GovukPublishingComponents::Engine, at: "/component-guide" if defined?(GovukPublishingComponents)
 
   scope format: false do
+    get '/templates/govuk_component/search.raw.html.erb', to: proc { [200, {}, ['This component has moved to the gem']] }
```

Comment:
@tijmenb just a quick thought on this - isn't it better if this line isn't included and an error gets thrown? Wouldn't we be more likely to spot it before deploy?

Reply:
Yeah, maybe just remove it. We better check the kibana logs for any calls to this template before deploying this PR.